tests/loggen/loggen.c

```diff
@@ -456,7 +456,7 @@ gen_messages(send_data_t send_func, void *send_func_ud, int thread_id, FILE *rea
           /* check rate every 0.1sec */
           long new_buckets;
 
-          new_buckets = (rate * diff_usec) / USEC_PER_SEC;
+          new_buckets = (long) ((rate * diff_usec) / USEC_PER_SEC);
```

Comment:
CC: @norberttak for #1987 :)

Reply:
@furiel : What do I change here ?

Reply:
I left the note for @norberttak because I believe these two requests will conflict on loggen.c. Looking at the changeset in #1987, it is possible that the rebase is not trivial, and this change needs to be integrated manually to the other request. But probably I am overcomplicating things now. The point is: you do not need to do any change here, unless #1987 will go in earlier than your PR. Nevertheless, the race is ON :)

Reply:
I addedd this modification to #1987 manually.